runtime/parachains/src/paras.rs

```diff
@@ -242,6 +245,26 @@ decl_module! {
 	/// The parachains configuration module.
 	pub struct Module<T: Trait> for enum Call where origin: <T as system::Trait>::Origin, system = system {
 		type Error = Error<T>;
+
+		/// Schedule a para to be initialized at the start of the next session.
+		#[weight = (1_000, DispatchClass::Operational)]
+		pub fn sudo_schedule_para_initialize(
```

Comment:
I would prefer this be in a separate module, because this is a low-level module that we shouldn't need to expose directly

Reply:
It should be in the `common` crate or a separate crate, though, not the `parachains` crate. my rationale is that the parachains-core layer should be agnostic as to _how_ and _when_ paras get registered, only that they do.

Reply:
moved to `common`